00-open-a-shapefile.Rmd

```diff
@@ -36,40 +37,36 @@ After completing this activity, you will:
 
  * Know the difference between point, line, and polygon vector elements.
  * Understand the differences between opening point, line and polygon shapefiles
-    in R.
- * Understand the components of a spatial object (in R).
+    in `R`.
+ * Understand the components of a spatial object in `R`.
 
 ###Things You'll Need To Complete This Lesson
 
-You will need the most current version of R or RStudio loaded on your computer 
-to complete this lesson.
-
+You will need the most current version of `R` and, preferably, RStudio loaded on
+your computer to complete this lesson.
 
-###R Libraries to Install:
+####R Libraries to Install:
 
 * **raster:** `install.packages("raster")`
-* **sp:** `install.packages("sp")`
 * **rgdal:** `install.packages("rgdal")`
+* **sp:** `install.packages("sp")`
 
 
-<a href="http://neondataskills.org/R/Packages-In-R/" target="_blank">More on Packages in R - Adapted from Software Carpentry.</a>
+* <a href="http://neondataskills.org/R/Packages-In-R/" target="_blank">More on Packages in R - Adapted from Software Carpentry.</a>
 
 ##Data to Download
 
-Download the vector files for the Harvard Forest dataset:
-
 <a href="http://files.figshare.com/2387960/boundaryFiles.zip" class="btn btn-success"> 
-DOWNLOAD Harvard Forest Shapefiles</a>
+DOWNLOAD Harvard Forest Field Site Shapefiles</a>
 
-The data used in this tutorial were collected at Harvard Forest which is
-a the National Ecological Observatory Network field site 
-<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank"> More about the NEON Harvard Forest field site</a>.
+The data used in this tutorial were collected  at the
+[National Ecological Observatory Network's](http://www.neoninc.org/)
+[Harvard Forest field site](http://www.neoninc.org/science-design/field-sites/harvard-forest) 
```

Comment:
Good question! IN web link context, an external link is one that is a DIFFERENT WEBSITE (base url) than the website the user is currently on. Essentially it is taking the user to a different site. We've gotten feedback that users would prefer new tabs when the links are external so they can keep the lessons open while working. I think it's best to be consistent with this throughout.

In some cases it will be confusing because for instance the data-lessons URL isn't the final URL - neondataskills is. BUT lucky for us, if we use {{ site.baseurl }} the base url will switch to NDS when we move the lessons over. 

Make sense?